core/domain/question_services.py

```diff
@@ -196,6 +196,26 @@ def get_question_skill_links_of_skill(skill_id):
     return question_skill_link_models
 
 
+def get_question_skill_links_of_question(question_id):
+    """Returns a list of QuestionSkillLinkModels of
+    a particular question ID.
+
+    Args:
+        question_id: str. ID of the question.
+
+    Returns:
+        list(QuestionSkillLinkModel). The list of question skill link
+        domain objects that are linked to the question ID or an empty list
```

Comment:
I don't think the domain objects for QuestionSkillLink are being returned currently. Can you change the returned object to be a list of QuestionSkillLink domain objects instead of datastore objects?

Reply:
fixed